criu/cr-service.c

```diff
@@ -285,6 +285,31 @@ int exec_rpc_query_external_files(char *name, int sk)
 
 static char images_dir[PATH_MAX];
 
+static int resolve_images_dir_path(char images_dir_path[PATH_MAX],
```

Comment:
I think for the compiler it will be just a char pointer. There is no guarantee about a size of images_dir_path.